.azure-pipelines-templates/common.yml

```diff
@@ -6,6 +6,10 @@ jobs:
     ${{ insert }}: ${{ parameters.env }}
 
     steps:
+      - script: |
+          sudo rm -rf build
```

Comment:
This comment could be wrong but should the delete be for . ?

Reply:
No, that's the git repo and will be 1. untouched by the tests and 2. reset correctly by DevOps, selectively nuking build/ is the right thing to do.

Reply:
This cleanup step happens before the checkout so deleting `.` would work too but as @achamayou said, the test doesn't touch anything apart from `build/` (`build/workspace/` in fact) so this should be sufficient.